resources/META-INF/plugin.xml

```diff
@@ -8,20 +8,35 @@
 
   <category>Custom Languages</category>
 
-  <version>18.3</version>
+  <version>18.4</version>
```

Comment:
18.4 is the current version. I thought I heard 19 mentioned.

Reply:
Yup; I'm updating here to the current released version (we only rev'd to 18.4 on the release_18 branch, so this is moving that info to master).

@pq will likely follow up with a PR to rev to 19.0

Reply:
👍